lib/service/invoices.go

```diff
@@ -235,7 +263,7 @@ func (svc *LndhubService) HandleSuccessfulPayment(ctx context.Context, invoice *
 	return err
 }
 
-func (svc *LndhubService) AddOutgoingInvoice(ctx context.Context, userID int64, paymentRequest string, decodedInvoice *lnrpc.PayReq) (*models.Invoice, error) {
+func (svc *LndhubService) AddOutgoingInvoice(ctx context.Context, userID int64, paymentRequest string, decodedInvoice *lnrpc.PayReq, keysend bool) (*models.Invoice, error) {
```

Comment:
i think we can maybe add struct that would contain both *lnrpc.PayReq and keysend, just wanted to check if it makes sense from domain perspective?

Reply:
I think it makes sense, yes. Also in the future we might have bolt12 or other extensions of the PayReq. So you can implement it.